CPDShell/shell.py

```diff
@@ -97,26 +98,25 @@ class CPDShell:
 
     def __init__(
         self,
+        scenario: Scenario,
         data: Iterable[float | numpy.float64] | LabeledCPData,
         cpd_algorithm: Optional["Algorithm"] = None,
-        scrubber_class: type[Scrubber] = Scrubber,
+        scrubber: AbstractScrubber = LinearScrubber(),
     ) -> None:
         """CPDShell object constructor
 
         :param: data: data for detection of CP
-        :param: CPDalgorithm: CPD algorithm, that will search for change points
-        :param: scrubber_class: class of preferable scrubber for splitting data into parts
+        :param: cpd_algorithm: CPD algorithm, that will search for change points
+        :param: scrubber: scrubber object for splitting data into parts
         """
         self._data: Iterable[float | numpy.float64] | LabeledCPData = data
-        scrubber_class = scrubber_class if scrubber_class is not None else Scrubber
         arg = 5
         cpd_algorithm = (
             cpd_algorithm if cpd_algorithm is not None else GraphAlgorithm(lambda a, b: abs(a - b) <= arg, 2)
         )
         self.cpd_core: CPDCore = CPDCore(
-            scrubber_class(Scenario(10, True), data.raw_data if isinstance(data, LabeledCPData) else data),
-            cpd_algorithm,
-        )  # if no algo or scrubber was given, then some standard
+            scenario, data.raw_data if isinstance(data, LabeledCPData) else data, scrubber, cpd_algorithm, 10
```

Comment:
Strange default value of `scrubber_data_size`. It is less than the window size in almost all cases.